dgraph/cmd/zero/tablet.go

```diff
@@ -176,12 +176,37 @@ func (s *Server) movePredicate(predicate string, srcGroup, dstGroup uint32) erro
 		Predicate: predicate,
 		SourceGid: srcGroup,
 		DestGid:   dstGroup,
-		TxnTs:     ids.StartId,
+		ReadTs:    ids.StartId,
+		SinceTs:   0,
+	}
+
+	// Move the predicate. Commits on this predicate is not blocked yet. Any data after TxnTs
```

Comment:
nit: Commits on this predicate _are_ not blocked yet. Any data above the _read_ts_